src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const longTerm = 7;
+  const shortTerm = 3;
+  const bigDiscount = 50;
+  const smallDiscount = 20;
+  const price = 40;
+
+  if (days >= longTerm) {
+    return days * price - bigDiscount;
+  }
+
+  if (days >= shortTerm) {
+    return days * price - smallDiscount;
+  }
```

Comment:
The condition `if (days >= shortTerm)` checks for short-term rentals. Make sure this logic is intended to apply only when the long-term condition is not met.